scopes/angular/eslint-config-bit-angular/component.json

```diff
@@ -0,0 +1,40 @@
+{
+  "componentId": {
+    "name": "angular",
```

Comment:
this is a wrong component id
This file should be first auto generated by the `bit eject-conf` command.
(I see the eslint-react has a wrong name as well.. probably it was manual created as well)
I recommend to copy it aside, generate then re-adding missing stuff

Reply:
I actually renamed that but forgot to add it to the commit. I didn't know about eject-conf though, thanks. PR updated.